modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValidatorImpl.java

```diff
@@ -149,6 +151,24 @@ protected void validateDDMFormFieldValidationExpression(
 		}
 	}
 
+	private void _findDuplicatedDDMFormFieldReferences(
```

Comment:
@devpedrotavares please rename it to `_addDuplicatedDDMFormFieldReferences`

Reply:
Done!